streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -325,6 +372,7 @@ public void resume() {
                 throw new ProcessorStateException(format("%sError while deleting the checkpoint file", logPrefix), e);
             }
         }
+        initializeMetadata();
```

Comment:
This is also new -- we implicitly re-initialize the task on `resume()` -- was done "outside" before

Reply:
Do we really need to re-initialize committed offsets and task time during Resume?

Reply:
Yes, because in `StreamTask#suspend()` we call `closeTopology()` that calls `partitionGroup.clear()` that resets all times to `UNKNOWN`.